Android.Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+pipeline {
+	environment {
+		NODE_PATH="/opt/node-v16.3.0-linux-x64/bin"
+		VERSION = sh(returnStdout: true, script: "${NODE_PATH}/node -p -e \"require('./package.json').version\" | tr -d \"\n\"")
+		APK_SIGN_STORE = '/opt/android-keystore/android.jks'
+		APK_SIGN_STORE_PASS = credentials('apk-sign-store-pass')
+		PATH="${env.NODE_PATH}:${env.PATH}"
+		ANDROID_SDK_ROOT="/opt/android-sdk-linux"
+		ANDROID_HOME="/opt/android-sdk-linux"
+	}
+
+	agent {
+		label 'linux'
+	}
+
+	parameters {
+		booleanParam(name: 'PROD', defaultValue: false, description: 'Build for production')
+		booleanParam(
+			name: 'PUBLISH', defaultValue: false,
+			description: "Publish the app to Nexus and Google Play (when not in production mode, " +
+					     "it will publish to the internal testing track of the tutanota-test app on Play)"
+		 )
+	}
+
+	stages {
+		stage('Run Android tests') {
+			steps {
+				dir("${WORKSPACE}/app-android/") {
+					sh "JAVA_HOME=/opt/jdk1.8.0_112 ./gradlew test"
+				}
+			}
+		}
+
+		stage('Build android app: production') {
+			environment {
+				APK_SIGN_ALIAS = "tutao.de"
+				APK_SIGN_KEY_PASS = credentials('apk-sign-key-pass')
+			}
+			when {
+				expression { params.PROD }
+			}
+			steps {
+				echo "Building ${VERSION}"
+				sh 'npm ci'
+				sh 'node android.js -b release prod'
```

Comment:
Probably a good point to move android.js to buildSrc?

Reply:
I think better to leave it out because it runs `node dist` or `node make` directly. Might make sense to provide it the location of an already built webclient or something like that instead, but that could be an overcomplication. I would leave it for now